nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentManager.java

```diff
@@ -319,6 +356,186 @@ public void testCommentManagerType() {
         assertEquals(getType(), commentManager.getClass());
     }
 
+    @Test
+    public void shouldNotifyEventWhenCreateComment() {
+        DocumentModel domain = session.createDocumentModel("/", "domain", "Domain");
```

Comment:
Do you really need to create a Domain then the test document under it? Couldn't you just create the test doc under `/` (root document)?

Reply:
i agree i just keep the way that the comment are created in this tests (each test creates a comment under the `domain`)